src/transaction/boot_cl.c

```diff
@@ -5590,8 +5590,16 @@ boot_define_view_db_server (void)
     }
 
   sprintf (stmt,
-	   "SELECT [ds].[link_name], [ds].[host], [ds].[port], [ds].[db_name], [ds].[user_name], [ds].[properties], [ds].[owner].[name], [ds].[comment]"
-	   " FROM [%s] [ds] WHERE CURRENT_USER = 'DBA' OR CURRENT_USER = [owner].[name]", CT_DB_SERVER_NAME);
+	   "SELECT [ds].[link_name], [ds].[host], [ds].[port], [ds].[db_name], [ds].[user_name], [ds].[properties],"
+	   "       [ds].[owner].[name], [ds].[comment]"
+	   " FROM [%s] [ds] WHERE CURRENT_USER = 'DBA' "
+	   "  OR {[ds].[owner].[name]} SUBSETEQ (SELECT SET{CURRENT_USER} + COALESCE(SUM(SET{[t].[g].[name]}), SET{})"
+	   "        FROM [%s] [u], TABLE([groups]) AS [t]([g]) WHERE [u].[name] = CURRENT_USER)"
+	   "  OR {[ds]} SUBSETEQ ( SELECT SUM(SET{[au].[class_of]}) FROM [%s] [au] WHERE {[au].[grantee].[name]} "
+	   "        SUBSETEQ ( SELECT SET{CURRENT_USER} + COALESCE(SUM(SET{[t].[g].[name]}), SET{}) "
+	   "                   FROM [%s] [u], TABLE([groups]) AS [t]([g])"
+	   "                   WHERE [u].[name] = CURRENT_USER) AND [au].[auth_type] = 'SELECT')", CT_DB_SERVER_NAME,
+	   CT_USER_NAME, CT_CLASSAUTH_NAME, CT_USER_NAME);
```

Comment:
Change the creation syntax of DB_SERVER VIEW
Changed so that members of the same group can also access it.